### ISSUE TITLE
debootstrap command should allow to pass extra component

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#310]
Originally reported of Google Code with ID 254.

```
Would be fine if create script could pass a --components to debootstrap, allowing to enable extra "component" (ex universe for ubuntu releases).
Tested locally, seems to be a simple fix, something like :
 
  # INCLUDE will be empty if EXTRA_PKGS is null/empty, otherwise we
  # build the full parameter format from it
  INCLUDE=${EXTRA_PKGS:+"--include=$EXTRA_PKGS"}
  COMPONENTS=${EXTRA_COMPONENTS:-"main"}
  debootstrap \
    --arch "$DPKG_ARCH" \
    $INCLUDE \
    --components=$COMPONENTS \
    "$SUITE" $TMPDIR $MIRROR

should be enough.
```
Originally added on 2012-08-04 08:12:37 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-29 16:32:41 +0000 UTC.

Added Labels: Type-Enhancement Component-instances
Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released in version 0.12.
```
Originally added on 2012-10-30 12:09:03 +0000 UTC.

Changed State: Released
